packages/next-codemod/bin/next-codemod.ts

```diff
@@ -33,13 +33,24 @@ const program = new Command(packageJson.name)
   .helpOption('-h, --help', 'Display this help message.')
   .option('-f, --force', 'Bypass Git safety checks and forcibly run codemods')
   .option('-d, --dry', 'Dry run (no changes are made to files)')
-  .option('-p, --print', 'Print transformed files to your terminal')
+  .option(
+    '-p, --print',
+    'Print transformed files to stdout, useful for development'
```

Comment:
> Print transformed files to stdout, useful for development

Got it from jscodeshift: https://github.com/facebook/jscodeshift?tab=readme-ov-file#usage-cli